mono/metadata/exception-internals.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _MONO_METADATA_EXCEPTION_INTERNALS_H_
+#define _MONO_METADATA_EXCEPTION_INTERNALS_H_
+
```

Comment:
`#include <mono/metadata/object.h>` to pickup `MonoArray`, `MonoException` typedef.
`#include <mono/utils/mono-error.h>` to pick up `MonoError`